src/libraries/System.IO.Ports/tests/SerialPort/GetPortNames.cs

```diff
@@ -20,6 +20,7 @@ public class GetPortNames : PortsTest
         /// </summary>
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoNorServerCore))]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34490", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [SkipOnPlatform(TestPlatforms.Android, "Not supported on Android")]
```

Comment:
This project is disabled in the top level file in the "common Android failures" section. Is this needed?

I do not think we care about supporting System.IO.Ports on Android as a whole.

Reply:
That's right. These changes were made before I disabled the System.IO.Tests in the tests.proj.